src/pages/sessions/evaluation.tsx

```diff
@@ -131,9 +129,8 @@ function Evaluation(): React.ReactElement {
           return null
         }
 
-        const { activeInstances, feedback, confusionTS, sessionStatus, instanceSummary } = extractInstancesFromSession(
-          session
-        )
+        const { activeInstances, sessionStatus, instanceSummary } = extractInstancesFromSession(session)
```

Comment:
Pulled out feedbacks and confusionTS from the helper function, as that is named and scoped to work on question instances and has nothing to do with the other functionality.